src/components/countrycard/__test__/CountryCard.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import CountryCard from '../CountryCard';
+import store from '../../../redux/store';
+
+test('It should render element with text Learn More', async () => {
+  render(
+    <Provider store={store}>
+      <CountryCard />
+    </Provider>,
+  );
+  const headingElement = screen.getByText('Learn More');
+  expect(headingElement).toBeInTheDocument();
+});
+
+test('It should render props text', async () => {
+  render(
+    <Provider store={store}>
+      <CountryCard name="Spain" />
+    </Provider>,
+  );
+  const headingElement = screen.getByText('Spain');
+  expect(headingElement).toBeInTheDocument();
```

Comment:
- [ ] There are failed tests, Please fix them.

![image](https://user-images.githubusercontent.com/68796232/217881607-03284163-953a-41a3-bb14-57e417f7e0e6.png)

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library), Please add integration tests using react testing library by mocking the access to the API and the connection to the Redux Store.